src/System.Net.NetworkInformation/tests/FunctionalTests/NetworkInterfaceBasicTest.cs

```diff
@@ -132,6 +178,56 @@ public void BasicTest_GetIPInterfaceStatistics_Success()
         }
 
         [Fact]
+        [PlatformSpecific(PlatformID.Linux)]
+        public void BasicTest_GetIPInterfaceStatistics_Success_Linux()
+        {
+            // This API is not actually IPv4 specific.
```

Comment:
I forgot to remove the comment. When I've ported S.N.NetworkInformation to GitHub I discovered the test and the behavior for GetIPv4Statistics(). At that time we decided to remove the IPv4 statistics API from the surface since we cannot support it on Windows (5a67678925c2f2731fdb55eb3c88a287149e8db4) in ref/System.Net.NetworkInformation.cs.

In the original test, the comment made sense. Now it doesn't (in either Win/*NIX test):

```C#
        public void BasicTest_GetIPv4InterfaceStatistics_Success()
        {
            // This API is not actually IPv4 specific.
            foreach (NetworkInterface nic in NetworkInterface.GetAllNetworkInterfaces())
            {
                IPv4InterfaceStatistics stats = nic.GetIPv4Statistics();
```